File: src/server/l5idapi/active/models/keyset.go
package models

import l5idapi_models "github.com/SuperTavor/Puniemu/src/server/l5idapi/models"

type KeySet struct {
	//ID for the player's client.
	UDKey l5idapi_models.Key `json:"udkey"`
	//GDKeys are IDs of the saves on the server. They can be retrieved with the UDKey.
	GDKeys []l5idapi_models.Key `json:"gdkeys"`
}

File: Src/Server/L5ID/Requests/Active/DataClasses/KeySet.cs
using Newtonsoft.Json;
using Puniemu.Src.Server.L5ID.DataClasses;

namespace Puniemu.Src.Server.L5ID.Requests.Active.DataClasses
{
    public struct KeySet
    {
        //ID for the player's client
        [JsonProperty("udkey")]
        public Key UDKey { get; set; }

        //Save file IDs associated with the udkey
        [JsonProperty("gdkeys")]
        public List<Key> GDKeys { get; set; }
    }
}
